bw_client.py

```diff
@@ -112,40 +112,50 @@
 
     bwClient = TCPClient()
     bwClient.connect(ip, port)
+    aliveCounter = 0
     while 1:
+        if not inputQueue.empty():
+            if not bwClient.isConnected:
```

Comment:
This call will also send the `keep-alive`, didn't it?
But now every 10ms instead of 100ms?

Reply:
That's correct, if a package is in the queue, a keep alive is sent every 10ms and if it is acknowledged, the package is sent.

But if no package is in the queue, the keep alive is only sent every second in the first else part.

In my case, most of the time, no package is in the queue, so it's working fine and sending the keep alive message every second; but if a package is in the queue, it is transmitted as fast as possible (every 10 ms to avoid cpu overload).